### PR TITLE
Change achievements label

### PR DESCRIPTION
**Pull request by jst-cyr**
Updated the 'Achievements' box with a new Card display to make it show a bit better on backgrounds, and also updated the title to 'Product Recommendations' based on feedback from the marketing team.

### CODE REVIEW

File: src/components/ui/HexagonCollection/HexagonCollection.tsx [hunk 1]
```diff
@@ -27,60 +27,67 @@ export const HexagonCollection: FC<HexagonCollectionProps> = ({ classStyles }) =
...
-      )}
-    </Box>
+    <Card>
+      <CardHeader>
+        <Heading size="md" variant={'achievements'}>
+          <Center>Product Recommendations</Center>
+        </Heading>
+      </CardHeader>
+      <CardBody>
+        {/* <Text fontSize={['lg', '2xl']} fontWeight="bold" textAlign="center" mb={[2, 10]} variant={'achievements'}>
```
Comment (thomas-desmond) [inline src/components/ui/HexagonCollection/HexagonCollection.tsx:37]: Can we remove this commented out seciton

**Review by thomas-desmond — CHANGES_REQUESTED** [review]
If we remove the commented section this should be good to go. I like using the <Card>, it helps the hexagons stand out a lot more.

**Comment by jst-cyr** [comment]
> If we remove the commented section this should be good to go. I like using the , it helps the hexagons stand out a lot more.

Removed the commented section. Good catch!

**Review by thomas-desmond — APPROVED** [review]

**Comment by thomas-desmond** [comment]
Looks even better with the UI changes

**Comment by jst-cyr** [comment]
I agree it looks better so merging this now, but we need to address some contrast issues with the background images.

**Pull request merged**